docs/source/format/how_manager.rst

```diff
@@ -0,0 +1,204 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+================================================
+How Drivers and the Driver Manager Work Together
+================================================
+
+.. note:: This document focuses on drivers/applications using the C API
+          definitions in adbc.h.  That means C/C++/Python/Ruby, and possibly
+          C#/Go.
```

Comment:
Shouldn't *all* drivers be using the definitions in adbc.h?

Reply:
Not in Rust, Go, etc.?

Reply:
Hmm, maybe we should reword this then? drivers/applications that provide C shared objects?

Reply:
It's not necessarily about shared objects either. It's about drivers that use those API definitions/that ABI.

Reply:
If they are providing a loadable driver for ADBC, it should be using those API definitions/that ABI. Otherwise it wouldn't be compatible, right?

Reply:
But you don't have to have a shared object. You could provide a static library for C/C++ users. You could (as R does) embed multiple driver definitions in one library and not provide the Adbc* functions in the first place.

Reply:
It's not related to whether a shared object is involved in the first place

Reply:
Hmm. fair, just seems like there's a better way to describe it but I can't think of one at the moment

Reply:
Changed to

```
.. note:: This document focuses on drivers/applications that implement or
          consume the C API definitions in adbc.h.  That includes C/C++,
          Python, and Ruby; and possibly C#, Go, and Rust (when implementing
          or consuming drivers via FFI).
```

Reply:
also, Java _never_ uses the header